db.cosi

```diff
@@ -166,7 +166,7 @@ meatpod                 IN CNAME        bacon
 metapod                 IN CNAME        bacon
 defcon                  IN CNAME        bacon
 xen                     IN CNAME        bacon
-beep                    IN CNAME        ziltoid
+beep                    IN CNAME        kasper
```

Comment:
`beep` is a dead meme and could be removed. It was a small program running on old Ziltoid, before the bios failures. It would run the beeper speaker and sounded like an alarm, pranking sys-admins.

Reply:
Forgot this PR was still open, removed beep.